lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescaleset.py

```diff
@@ -509,6 +517,9 @@ def exec_module(self, **kwargs):
         if not self.virtual_network_resource_group:
             self.virtual_network_resource_group = self.resource_group
 
+        if self.capacity > 100:
+            self.single_placement_group = False
+
```

Comment:
could you try one scenario:
(1) create VMSS with **single_placement_group: yes**
(2) change **single_placement_group: no**
just need to make sure if it will be updated / can be updated